source/preprocessor.tex

```diff
@@ -1194,6 +1267,51 @@
 \tcode{\#\#}
 operators is unspecified.
 
+\pnum
+\begin{example}
+To illustrate the rules for creating character string literals
```

Comment:
We can probably do without the "To illustrate" here and below now that they're in the right places.

Reply:
Fixed.